### DISCUSSION TITLE
Please tell me your favorite Anime Character!

### DISCUSSION THREAD

**Discussion by yomaytk** [NAIST-SE/AttractingContributors#192] [General]
Tell me whatever your favorite anime character is!
I'm expecting a pull request.
https://github.com/yomaytk/AttractingContributors-AnimeCharacter

**Comment by naistclass** [comment]
I have sent a Pull Request. Please check.
And will you help me?

https://github.com/naistclass/AttractingContributors-FavoriteFood

**Reply by yomaytk** [reply]
Thank you for your pull request!
I added favorite food to your repository!

**Comment by TakumiHiraide** [comment]
I have added your repository a character.
Please help me!!
https://github.com/TakumiHiraide/My-favorite-country

**Reply by yomaytk** [reply]
Thank you for your pull request!
I added favorite country to your repository!

**Comment by pgDora56** [comment]
I have sent a PR, please check it.

I need your help! Tell me your favorite youtuber!
https://github.com/pgDora56/AttractingContributors-Youtuber

**Reply by yomaytk** [reply]
Thank you for your pull request!
I added favorite youtuber to your repository!

**Comment by t-matsumoto4758** [comment]
Added!
please tell me your hometown!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Reply by yomaytk** [reply]
Thank you for your pull request!
I added my hometown to your repository!

**Comment by xincanfeng** [comment]
Added!
please tell me your favorite color!
https://github.com/FengXincan/AttractingContributors-color

**Reply by yomaytk** [reply]
Thank you for your pull request!
I added favorite color to your repository!

**Comment by veekay7** [comment]
Just added a chara to your repo.
Please tell me the country/countries you wanna go to in the future.
Cheers!

https://github.com/VantelinLuca/AttractingContributors-WishCountry

**Comment by hernessteve** [comment]
I like Levi a lot! Question to professionals - is it worth to install Cinema apk in order to watch anime? [https://www.firesticktricks.com/cinehub-apk.html](https://www.firesticktricks.com/cinehub-apk.html ) here is written that you can find almost everything there. Is it right?
